test/algorithms/relational_operations/overlaps/test_overlaps.hpp

```diff
@@ -42,11 +45,25 @@ void test_geometry(std::string const& wkt1,
     detected = bg::overlaps(geometry2, geometry1);
 
     BOOST_CHECK_MESSAGE(detected == expected,
-        "overlaps: " << wkt1
-        << " with " << wkt2
+        "overlaps: " << wkt2
```

Comment:
I thought it's a mistake since the order of geometries for which the algorithm could fail to provide the correct result is reversed. So it seems to be more intuitive to output the WKTs in the order for which the algorithm is failing. But now I see that the original order is used in `test_equals`, `test_overlaps` and `test_intersects`) but not in `test_touches`. Furthermore, if the original order is printed out it should probably be easier to find the line where test case is implemented in the file. Do you think I should revert this change?